### PR TITLE
support sticky connection

### PR DESCRIPTION
**Pull request by CodingSinger**
support the sticky connection

### CODE REVIEW

File: cluster/cluster_impl/base_cluster_invoker_test.go [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cluster_impl
+
+import (
+	"context"
```
Comment (AlexStocks) [inline cluster/cluster_impl/base_cluster_invoker_test.go]: pls split import to 3 blocks as other files.

Comment (CodingSinger) [reply cluster/cluster_impl/base_cluster_invoker_test.go]: done~

**Comment by skyitachi** [comment]
should stickyInvoker check available?

**Comment by CodingSinger** [comment]
> should stickyInvoker check available?

yes, i have added this logic.

**Pull request merged**